### PR TITLE
Update Level Up docs

### PR DESCRIPTION
**Pull request by JihwanEom**
### Summary

- Align command expression to `command`
- Fix typos

### Checklist

- [ ] I have added unit tests to cover my changes.​
- [ ] I have added integration tests to cover my changes.​
- [ ] I have added the description of my changes into [CHANGELOG](https://github.com/openvinotoolkit/datumaro/blob/develop/CHANGELOG.md).​
- [ ] I have updated the [documentation](https://github.com/openvinotoolkit/datumaro/tree/develop/docs) accordingly

### License

- [ ] I submit _my code changes_ under the same [MIT License](https://github.com/openvinotoolkit/datumaro/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [ ] I have updated the license header for each file (see an example below).

```python
# Copyright (C) 2023 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

**Review by cih9088 — APPROVED** [review]

**Review by sooahleex — APPROVED** [review]

File: docs/source/docs/level-up/intermediate_skills/10_data_generation.rst [hunk 1]
```diff
@@ -40,6 +40,6 @@ We thus not need to concern about any privacy issues.
     ``GEN_IMG_COUNT`` is an integer that indicates the number of images to be generated. (e.g. ``count=300``)
     ``GEN_IMG_SHAPE`` is a tuple representing the shape of generated images as (width, height) (e.g. ``shape=(240, 180)``)
 
-Congratulations! You complete reading all Datumaro level-up documents for the intermediate skills.
+Congratulations! You completed reading all Datumaro level-up documents for the intermediate skills.
 
-.. [FractalDB] Can Vision Transformers Learn without Natural Images?, Kodai Nakashima, et. al., 2013.
+.. [FractalDB] Can Vision Transformers Learn without Natural Images?, Kodai Nakashima, et. al., 2021.
```
Comment (vinnamkim) [inline docs/source/docs/level-up/intermediate_skills/10_data_generation.rst]:
```suggestion
.. [FractalDB] Nakashima, Kodai, et al. "Can vision transformers learn without natural images?." Proceedings of the AAAI Conference on Artificial Intelligence. Vol. 36. No. 2. 2022.
```
I recently used MLA format for the reference somewhere. Please change it to MLA format.

File: docs/source/docs/level-up/basic_skills/01_project_manipulation.rst [hunk 2]
```diff
@@ -2,10 +2,10 @@
 Level 1: Project Manipulation
 =============================
 
-Projects are intended for complex use of Datumaro CLI commands.
-If you want to know more about the concept and meaning of the project, please see :ref:`Projects`.
-You might be able to skip this page, if you plan to do various dataset management tasks using the Datumaro Python API rather than CLI commands.
-However, even for the Python API, you have to get back to the project if you need dataset versioning.
+Projects are designed to facilitate the use of Datumaro CLI commands for complex tasks.
+If you need more information about the project's concept and meaning, please refer :ref:`Projects`.
```
Comment (vinnamkim) [inline docs/source/docs/level-up/basic_skills/01_project_manipulation.rst]:
```suggestion
If you need more information about the project's concept and meaning, please refer to :ref:`Projects`.
```

File: docs/source/docs/level-up/basic_skills/02_dataset_download.rst [hunk 3]
```diff
@@ -29,7 +29,7 @@ Which datasets are available?
 
       datum download describe [--report-format {text,json}] [--report-file REPORT_FILE]
 
-How to download datasets?
+How can we download datasets?
 =========================
```
Comment (vinnamkim) [inline docs/source/docs/level-up/basic_skills/02_dataset_download.rst]: This `=` sequence must be the same length as the phrase immediately above.

**Review by vinnamkim — CHANGES_REQUESTED** [review]

File: docs/source/docs/level-up/basic_skills/02_dataset_download.rst (hunk 3 above)
Comment (JihwanEom) [reply docs/source/docs/level-up/basic_skills/02_dataset_download.rst]: Reflected, thank you for comment!

**Review by vinnamkim — APPROVED** [review]
LGTM.

**Pull request merged**